### PR TITLE
Building page

### PR DESCRIPTION
**Pull request by Salvador-ON**
All the containt of my page is generated by javascript (navbar, main-container with all the info and the footer)

each content ( home,donut menu and order) are in different files and imported to  the index js.

Then with webpack is generated the main.js that works with my html page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "restaurantpage",
+  "version": "1.0.0",
+  "description": "Restaurant Page with JS",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (bellom) [inline package.json:7]: I can't build the project with webpack.
You missed the **project-specific** section in the pathwright where you have to add the necessary webpack scripts to build the project.
image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/


1. We should have all necessarily used packages that should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) **in our case we are missing `webpack-dev-server`**
Kindly add `webpack-dev-server`.

2. We should be able to run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) in our own case here, we are missing it so I can't run `npm run build ` to generate `dist folder`, I should be able to run `npm run build` to generate `dist folder` at my end, currently using the one created by you, it shouldn't be like that.
kindly add these two scripts `build` and `development`

Comment (Salvador-ON) [reply package.json:7]: Thanks very much for the feedback. I appreciate it

Comment (Salvador-ON) [reply package.json:7]: i added the webpack-dev-server

Comment (Salvador-ON) [reply package.json:7]: i added the build script

Comment (Salvador-ON) [reply package.json:7]: i added the development script

**Comment by Salvador-ON** [comment]
i added the webpack-dev-server package and the build and development script

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurantpage",
+  "version": "1.0.0",
+  "description": "Restaurant Page with JS",
+  "private": true,
+  "scripts": {
+    "start:dev": "webpack-dev-server",
+    "build": "webpack",
+    "development": "webpack --env dev",
+    "start": "webpack-dev-server --open",
```
Comment (bellom) [inline package.json:10]: we only one `start` here, kindly remove the one on line 7.
update before merging.

**Pull request merged**